File: LicentaUI/Models/ReviewBookModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LicentaUI.Models
{
    public class ReviewBookModel
    {
        public string ID { get; set; }

        public string IdUser { get; set; }

        public string IdBook { get; set; }

        public int? Grade { get; set; }

        public string Review { get; set; }

        public Status Status { get; set; }
    }
}

File: LicentaAngular/src/app/models/review-book-model.ts
import { Status } from './status';
import { Genre } from './genre';

export interface ReviewBookModel {
    idReview: string;
    idUser: string;
    username: string;
    idBook: string;
    grade: number;
    review: string;
    status: Status;
    book: {
        title: string;
        author: string;
        description: string;
        prequeslId: string;
        sequelId: string;
        genre: Genre;
        picture: string;
    }
}
